src/main/java/marquez/client/MarquezClient.java

```diff
@@ -322,10 +322,8 @@ static Version get() {
       final Properties properties = new Properties();
       try (final InputStream stream =
           MarquezClient.class.getClassLoader().getResourceAsStream(CONFIG_PROPERTIES)) {
-        if (stream != null) {
```

Comment:
_Minor_: What spotbug rule was this `null` check breaking?

Reply:
It complained about Redundant condition check.

Reply:
Redundant nullcheck of stream, which is known to be non-null in marquez.client.MarquezClient$Version.get()